frontend/src/routes/manage/Video/index.tsx

```diff
@@ -175,7 +175,7 @@ const EventTable: React.FC<EventTableProps> = ({ events, vars }) => {
                 },
             },
             "& > tbody": {
-                "& > tr:hover": {
+                "& > tr:hover, tr:focus-within": {
```

Comment:
I think you are missing a `& >` here? It seems to work anyway. Mh. I have the strong feeling that most `&` I used are not necessary :D 

```suggestion
                "& > tr:hover, & > tr:focus-within": {
```